.travis.yml

```diff
@@ -11,7 +11,6 @@ cache:
     - $HOME/.cache # includes bower's cache
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
Upon closer look, it seems to refer to the [1.4.0 release of Hubspot Tether](https://github.com/HubSpot/tether/releases/tag/v1.4.0)